tests/integration/test_system_app_creator.py

```diff
@@ -0,0 +1,37 @@
+from .common import random_str
+import time
+
+
+def test_system_app_creator(admin_mc, admin_system_pc):
+    client = admin_mc.client
+    provider_name = random_str()
+    access = random_str()
+    secret = random_str()
+    globaldns_provider = \
+        client.create_global_dns_provider(
+                                         name=provider_name,
+                                         rootDomain="example.com",
+                                         route53ProviderConfig={
+                                             'accessKey': access,
+                                             'secretKey': secret})
+    app = wait_for_system_app(
+        admin_system_pc.client,
+        "systemapp-"+globaldns_provider.name)
+    # the creator id of system app won't be listed in api
+    users = client.list_user(id=app.creatorId)
+    assert len(users) == 0
```

Comment:
I get we can't assert that the system account of local cluster is the creator of the app, but we can assert that the creator of this gdns_provider (admin in this case) is not the creator of the app.